src/component/project/ProjectEnvironment/ProjectEnvironment.tsx

```diff
@@ -129,10 +132,21 @@ const ProjectEnvironmentList = ({ projectId }: ProjectEnvironmentListProps) => {
     };
 
     const genLabel = (env: IProjectEnvironment) => (
-        <>
-            <code>{env.name}</code> environment is{' '}
-            <strong>{env.enabled ? 'enabled' : 'disabled'}</strong>
-        </>
+        <div className={commonStyles.flexRow}>
+            <code>
+                <StringTruncator
+                    text={env.name}
+                    maxLength={50}
+                    maxWidth="150"
+                />
+            </code>
+            {/* This is ugly - but regular {" "} doesn't work here*/}
```

Comment:
Maybe without the `flexRow`:

```
const genLabel = (env: IProjectEnvironment) => (
    <p>
        <code>
            <StringTruncator
                text={env.name}
                maxLength={50}
                maxWidth="150"
            />
        </code>{' '}
        environment is{' '}
        <strong>{env.enabled ? 'enabled' : 'disabled'}</strong>
    </p>
);
```

Reply:
Yes, that will work. But it causes a misalignment:

<img width="447" alt="Skjermbilde 2022-03-11 kl  12 12 45" src="https://user-images.githubusercontent.com/16081982/157856580-5018cdf4-5074-4482-a781-9052fb078191.png">
 
 I think we can review this later.